inst/csv/replacementPatterns.csv

```diff
@@ -485,22 +485,38 @@ netezza,LOG10(@expression),LOG(@expression)
 netezza,"ISNULL(@a,@b)","COALESCE(@a,@b)"
 netezza,COUNT_BIG(@a),COUNT(@a)
 netezza,USE @schema;,SET search_path TO @schema;
+netezza,"IF OBJECT_ID('@table', 'U') IS NULL CREATE TABLE @table (@definition);",CREATE TABLE IF NOT EXISTS @table (@definition);
 netezza,"IF OBJECT_ID('@table', 'U') IS NOT NULL DROP TABLE @table;",DROP TABLE @table IF EXISTS;
 netezza,.dbo.,.
 netezza,CREATE TABLE #@table (@definition),CREATE TEMP TABLE @table (@definition)
 netezza,CREATE CLUSTERED INDEX @index_name ON @table (@variable);,CREATE INDEX @index_name ON @table (@variable);\nCLUSTER @table USING @index_name;
 netezza,CREATE UNIQUE CLUSTERED INDEX @index_name ON @table (@variable);,CREATE UNIQUE INDEX @index_name ON @table (@variable);\nCLUSTER @table USING @index_name;
 netezza,PRIMARY KEY NONCLUSTERED,PRIMARY KEY
 netezza,VARCHAR(MAX),VARCHAR(1000)
-netezza,FLOAT,NUMERIC
+netezza,FLOAT,"FLOAT(6)"
 netezza,WITH @a AS @b SELECT @c INTO #@d FROM @e;,CREATE TEMP TABLE @d\nAS\nWITH @a AS @b SELECT\n@c\nFROM\n@e;
 netezza,WITH @a AS @b SELECT @c INTO @d FROM @e;,CREATE TABLE @d\nAS\nWITH @a AS @b SELECT\n@c\nFROM\n@e;
 netezza,SELECT @a INTO #@b FROM @c;,CREATE TEMP TABLE @b\nAS\nSELECT\n@a\nFROM\n@c;
 netezza,SELECT @a INTO @b FROM @c;,CREATE TABLE @b AS\nSELECT\n@a\nFROM\n@c;
 netezza,#,
-netezza,"LEFT(@variable,@b)","STRLEFT(@variable,@b)"
-netezza,"RIGHT(@variable,@b)","STRRIGHT(@variable,@b)"
+netezza,"LEFT(@variable,@b)","SUBSTR(@variable, 0, @b+1)"
+netezza,"RIGHT(@variable,@b)","SUBSTR(@variable, length(@variable)-@b+1, @b)"
 netezza,SELECT TOP @([0-9]+)rows @a;,SELECT @a LIMIT @rows;
+netezza,"CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(@a, @b), @c), @d), @e), @f), @g)", @a || @c || @e || @g
+netezza,"CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(@a, @b), @c), @d), @e), @f), @g), @h), @i)", @a || @c || @e || @g || @i
+netezza,"CONCAT(CONCAT(@a, @b), '@c')","@a || @b || '@c'"
+netezza,"CONCAT(CONCAT(@a, '@b'), '@c')","@a || '@b' || '@c'"
+netezza,"CONCAT('@a', @b,'@c',@d)","'@a' || @b || '@c' || @d"
+netezza,"CONCAT(@a, @b)","@a || @b"
+netezza,"CONCAT(@a, '@b')","@a || '@b'"
+netezza,"CONCAT('@a', @b)","'@a' || @b"
+netezza,POWER(,POW(
+netezza,EOMONTH(@date),LAST_DAY(@date)
+netezza,CAST(@a AS INT), CAST(@a AS BIGINT)
+netezza,ROWCOUNT(), ROW_NUMBER()
+netezza,WHERE rn <= 100, LIMIT 100

```

Comment:
This rule is making too many assumptions:

1. that there is a variable called `rn` that represents the row number

2. that there is nothing after `rn <= 100` in the WHERE clause (e.g. it will fail when the query ends with `WHERE rn <= 100 AND x = 5`)

3. That 100 is somehow a significant number?

Note that there is already a rule that with translate SQL Server's TOP argument to a LIMIT: https://github.com/OHDSI/SqlRender/blob/master/inst/csv/replacementPatterns.csv#L507